cob_bringup/drivers/sick_lms1xx.launch

```diff
@@ -6,11 +6,11 @@
 	<arg name="name"/>
 
 	<!-- load parameter -->
-	<rosparam ns="$(arg name)" command="load" file="$(arg pkg_hardware_config)/$(arg robot)/config/$(arg name).yaml"/>
+	<rosparam ns="$(arg name)/driver" command="load" file="$(arg pkg_hardware_config)/$(arg robot)/config/$(arg name).yaml"/>
```

Comment:
Because private names are used and node is starting with the name "driver" (see line 12 of this file). I wanted to make it more compatible to sick_lms300.

I am pretty sure that only raw3-5 is using lms1xx. So says the [care-o-bot web page](http://wiki.ros.org/Robots/Care-O-bot/distribution) and I greped the whole workspace.